Competition2017/src/competition/subsystems/agitator/AgitatorSubsystem.java

```diff
@@ -0,0 +1,46 @@
+package competition.subsystems.agitator;
+
+import javax.inject.Singleton;
+
+import org.apache.log4j.Logger;
+
+import com.ctre.CANTalon.TalonControlMode;
+import com.google.inject.Inject;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+@Singleton
+public class AgitatorSubsystem extends BaseSubsystem{
+    private static Logger log = Logger.getLogger(AgitatorSubsystem.class);
+    
+    public final DoubleProperty motorPowerProperty;
+    public final XCANTalon agitatorMotor;
+    
+    @Inject
+    public AgitatorSubsystem(WPIFactory factory, XPropertyManager propManager){
+        log.info("Creating Agitator Subsystem");
+        agitatorMotor = factory.getCANTalonSpeedController(6);
+        agitatorMotor.setBrakeEnableDuringNeutral(true);
```

Comment:
This should be false, I think. I don't see a reason for it to prevent itself from spinning, and it could get in the way.